src/components/Notification/NotificationDrawer/NotificationDrawerToggle.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { OverlayTrigger } from '../../OverlayTrigger';
+import { Tooltip } from '../../Tooltip';
+import { Icon } from '../../Icon';
+import { Button } from '../../Button';
+
+const NotificationDrawerToggle = ({ hasUnreadMessages, onClick }) => {
+  const iconName = hasUnreadMessages ? 'bell' : 'bell-o';
+  const tooltip = <Tooltip id="tooltip">Notifications</Tooltip>;
+
+  return (
+    <OverlayTrigger
+      placement="bottom"
+      id="notifications-toggle-icon"
+      overlay={tooltip}
+    >
+      <Button onClick={onClick} bsStyle="link">
+        <Icon
+          name={iconName}
+          style={{ color: 'rgb(240, 240, 240)', marginTop: '20px' }}
```

Comment:
this shouldn't be added. Patternfly has styling for the toggle while inside the mastheads. This will override them.

Reply:
I mentioned this before,  there is a CSS problem with the toggle. thats why i separated the `StatefulDrawerWrapper` from the toggle. i created this toggle for the demonstration only, ill try to find a better solution until the CSS experts arrive

Reply:
I see. Adding it in the storybook code would be OK by me. If/when we have a masthead component it won't be necessary (I would hope).

Reply:
@gilad215 I believe adding the `nav-item-iconic` class to the Button will get us the correct patternfly styling without the need for the inline styles.